core/src/i18n/en.yaml

```diff
@@ -105,6 +105,7 @@ applications:
     provided-services: Provided Services and events
   title: Applications
 brokers:
+  description: <0>Service Broker</0> manages the lifecycle of the offered services.
```

Comment:
.... or: `<0>Service Broker</0> provides a set of managed services that a third party offers and maintains.` Here's the [referece](https://kyma-project-old.netlify.app/docs/components/service-catalog#architecture-resources).